source/renderer/zettlr-body.js

```diff
@@ -851,12 +851,9 @@ class ZettlrBody {
           level = [ h1, h2, h3, h4, h5, h6 ].join('.')
       }
 
-      cnt.append(
-        $('<a>').text(level + '. ' + entry.text)
-          .attr('data-line', entry.line)
-          .attr('href', '#')
-          .addClass('toc-link')
-      )
+      const template = document.createElement('template')
+      template.innerHTML = `<a href="#" class="toc-link" data-line="${entry.line}">${level}. ${entry.text}</a>`
+      cnt.append(template.content.cloneNode(true))
```

Comment:
As above, please comment on why you chose this approach!

Reply:
Please, see my comment above.

Reply:
So we only need the template-rendering function here as well, did I get this right …? 🧐

Reply:
Yes, I've updated all instances now.